### ISSUE TITLE
Can't install ProLif on COLAB

### ISSUE THREAD

**Issue by hima111997** [chemosim-lab/ProLIF#29]
Hello
I am trying to install ProLIF on COLAB using the installation instructions nut gave me an error with this command
!pip install git+https://github.com/chemosim-lab/ProLIF.git.
COLAB link: https://colab.research.google.com/drive/1dZEUxEDbQhLy-pt1lfC8L_rE8gsvhnJ8?usp=sharing

Could You please help me?
Thanks

**Comment by radifar** [comment]
Hi @hima111997 , I have take a look at your notebook. And after I tried running this
```
!git clone https://github.com/chemosim-lab/ProLIF.git
!pip install ProLIF/
```
It gave the same error. Then as suggested in the error message I run `python setup.py egg_info` To see the log information
```
!python ProLIF/setup.py egg_info
Traceback (most recent call last):
  File "ProLIF/setup.py", line 12, in <module>
    raise ValueError("ProLIF requires a version of RDKit >= 2020")
ValueError: ProLIF requires a version of RDKit >= 2020
```
Now we found the source of the problem. After a quick look on the RDKit installation process I can see that your RDKit came from rdkit channel which is outdated, it gave version 2017. I suggest that you install RDKit from [conda-forge channel ](https://anaconda.org/conda-forge/rdkit) as instructed [here](https://www.rdkit.org/docs/Install.html#cross-platform-under-anaconda-python-fastest-install).

I hope that solve your problem.

**Comment by hima111997** [comment]
thanks it worked
    بتاريخ الأربعاء، 8 أيلول 2021, 05:39:18 ص غرينتش+2، Muhammad Radifar ***@***.***> كتب:  
 
 


Hi @hima111997 , I have take a look at your notebook. And after I tried running this
!git clone https://github.com/chemosim-lab/ProLIF.git
!pip install ProLIF/

It gave the same error. Then as suggested in the error message I run python setup.py egg_info To see the log information
!python ProLIF/setup.py egg_info
Traceback (most recent call last):
  File "ProLIF/setup.py", line 12, in <module>
    raise ValueError("ProLIF requires a version of RDKit >= 2020")
ValueError: ProLIF requires a version of RDKit >= 2020

Now we found the source of the problem. After a quick look on the RDKit installation process I can see that your RDKit came from rdkit channel which is outdated, it gave version 2017. I suggest that you install RDKit from conda-forge channel  as instructed here.

I hope that solve your problem.

—
You are receiving this because you were mentioned.
Reply to this email directly, view it on GitHub, or unsubscribe.
Triage notifications on the go with GitHub Mobile for iOS or Android.
